### PR TITLE
262 automate calculations

### PR DESCRIPTION
**Pull request by marcellmueller**
https://app.zenhub.com/workspaces/ccbc---delivery-board-61c0c1204c8bcb001491c5f3/issues/bcgov/conn-ccbc-portal/262

### CODE REVIEW

File: app/components/Form/ApplicationForm.tsx [hunk 1]
```diff
@@ -177,12 +179,52 @@ const ApplicationForm: React.FC<Props> = ({
...
-      <Button variant="secondary" style={{ marginLeft: '20px' }}>
-        Save
-      </Button> */}
-    </FormBase>
+    </>
+  );
+
+  return (
+    <>
+      {isCalculatedPage ? (
```
Comment (marcellmueller) [inline app/components/Form/ApplicationForm.tsx:187]: @VB-at-Bis @AntBush what do you guys think about this? I wasn't sure if there is a cleaner way to go about using the `CalculatedForm` renderer. This way we only use it on the pages that use the calculations but it feels a bit messy somehow.

Comment (VB-at-Bis) [reply app/components/Form/ApplicationForm.tsx:187]: It looks good for me :) .. not messy at all

Comment (AntBush) [reply app/components/Form/ApplicationForm.tsx:187]: I think this way for one or two it's fine, but we'll definitely have to think of a different way to do this if we have many custom forms we want to display.

**Review by AntBush — APPROVED** [review]
Looks good to me!

File: app/components/Form/ApplicationForm.tsx (hunk 1 above)
Comment (matthieu-foucault) [reply app/components/Form/ApplicationForm.tsx:187]: Did you check how calculated fields are handled in CIIP and CIF. Adding some section-specific branching in the `ApplicationForm` page is not a maintainable solution. The rjsf way of doing this is to define custom fields for those items that require calculations

**Pull request merged**